data_collection/gazette/spiders/rj/rj_macae.py

```diff
@@ -0,0 +1,53 @@
+import re
+from datetime import date, datetime as dt
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RjMacaeSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3302403"
+    name = "rj_macae"
+    allowed_domains = ["macae.rj.gov.br"]
+    start_date = date(2020, 5, 22)
+
+    def start_requests(self):
+        yield scrapy.FormRequest(
+            url="https://sistemas.macae.rj.gov.br:840/diariooficial/index/listarajax",
+            method="POST",
+            formdata={
+                "periodode": self.start_date.strftime("%d/%m/%Y"),
+                "periodoate": self.end_date.strftime("%d/%m/%Y"),
+            },
+        )
+
+    def parse(self, response):
+        for data in response.json()["data"]:
+            gazette_code = data["DT_RowId"]
+            gazette_url = f"https://sistemas.macae.rj.gov.br:840/diariooficial/index/download?id={gazette_code}"
+
+            gazette_edition = data["edicao"]
+            gazette_edition_number = re.search(r"\d+", gazette_edition).group(0)
+
+            raw_gazette_date = data["publicacao"][0:10]
```

Comment:
Aqui não tem nada formalmente errado (até pq a lógica está implementada pra consumir a API e isso tende a ser bastante consistente), mas não costumamos usar fatiamento de strings, por ser uma solução muito rígida. 

```suggestion
            raw_gazette_date = re.search(r"\d{2}\/\d{2}\/\d{4}", data["publicacao"]).group()
```

Caso tenha feito desse jeito para garantir que a data coletada seja a primeira - visto que aparece duas vezes - também resolve do jeito que sugiro, visto que o primeiro match que é levado em consideração.